Makefile

```diff
@@ -298,6 +298,91 @@ update-k8s-patch-versions: ## Run update-patchversion script to generate new ver
 
 ##@ Lib helm
 .PHONY: update-lib-helm
-update-lib-helm: ## Update lib-helm
+update-lib-helm: ## Update lib-helm.
 	##~ Options: version=MAJOR.MINOR.PATCH
 	cd helm_lib/ && yq -i '.dependencies[0].version = "$(version)"' Chart.yaml && helm dependency update && tar -xf charts/deckhouse_lib_helm-*.tgz -C charts/ && rm charts/deckhouse_lib_helm-*.tgz && git add Chart.yaml Chart.lock charts/*
+
+##@ Build
+.PHONY: build
+set-build-envs:
+  ifeq ($(WERF_ENV),)
+  	export WERF_ENV=FE
+  endif
+  ifeq ($(WERF_CHANNEL),)
+ 		export WERF_CHANNEL=ea
+  endif
+  ifeq ($(DEV_REGISTRY_PATH),)
+ 		export DEV_REGISTRY_PATH=dev-registry.deckhouse.io/sys/deckhouse-oss
+  endif
+  ifeq ($(SOURCE_REPO),)
+ 		export SOURCE_REPO=https://github.com
+  endif
+  ifeq ($(GOPROXY),)
+ 		export GOPROXY=https://proxy.golang.org/
+  endif
+  ifeq ($(CI_COMMIT_TAG),)
+ 		export CI_COMMIT_TAG=$(shell git describe --abbrev=0 2>/dev/null)
+  endif
+  ifeq ($(CI_COMMIT_BRANCH),)
+ 		export CI_COMMIT_BRANCH=$(shell git branch --show-current)
+  endif
+  ifeq ($(CI_COMMIT_REF_NAME),)
+ 		export CI_COMMIT_REF_NAME=$(shell git rev-parse --abbrev-ref HEAD)
+ 	else
+		ifeq ($(CI_COMMIT_TAG),)
+			export CI_COMMIT_REF_NAME=$(CI_COMMIT_BRANCH)
+		else
+			export CI_COMMIT_REF_NAME=$(CI_COMMIT_TAG)
+		endif
+ 	endif
+  ifeq ($(CI_COMMIT_REF_SLUG),)
+ 		export CI_COMMIT_REF_SLUG=$(shell echo $(CI_COMMIT_BRANCH) | cut -c -63 | sed -E 's/[^a-z0-9-]+/-/g' | sed -E 's/^-*([a-z0-9-]+[a-z0-9])-*$$/\1/g')
+ 	endif
+  ifeq ($(DECKHOUSE_REGISTRY_HOST),)
+ 		export DECKHOUSE_REGISTRY_HOST=registry.deckhouse.io
+  endif
+	export WERF_REPO=$(DEV_REGISTRY_PATH)
+	export REGISTRY_SUFFIX=$(shell echo $(WERF_ENV) | tr '[:upper:]' '[:lower:]')
+	export SECONDARY_REPO=--secondary-repo $(DECKHOUSE_REGISTRY_HOST)/deckhouse/$(REGISTRY_SUFFIX)
+
+build: set-build-envs ## Build Deckhouse images.
+	##~ Options: FOCUS=image-name
+	werf build --parallel=true --parallel-tasks-limit=5 --platform linux/amd64 --report-path images_tags_werf.json $(SECONDARY_REPO) $(FOCUS)
+  ifeq ($(FOCUS),)
+    ifneq ($(CI_COMMIT_BRANCH),)
+				@# CI_COMMIT_REF_SLUG is a 'prNUM' for dev branches or 'main' for default branch.
```

Comment:
The comment is misleading because we are using CI_COMMIT_REF_SLUG as the branch name by default